tripal_chado/config/install/tripal.tripal_content_terms.chado_content_terms.yml

```diff
@@ -985,11 +985,11 @@ vocabularies:
                 description: 'a versioned dataset is a dataset with a particular release date or release number.'
     -   name: 'PBO'
         label: 'Plant Breeding Ontology'
-        url: 'https://agroportal.lirmm.fr/ontologies/PBO'
+        url: 'http://purl.obolibrary.org/obo/PBO'
         idSpaces:
             -   name: 'PBO'
-                description: 'Plant Breeding Ontology'
-                urlPrefix: 'https://agroportal.lirmm.fr/ontologies/PBO/PBO_{accession}'
+                description: 'Plant Breeding Ontology (PBO): an ontology for the plant breeding community which captures more than 2200 entries where 80 represent the core terms.'
+                urlPrefix: 'http://purl.obolibrary.org/obo/PBO/PBO_{accession}'
```

Comment:
I tried to check thie upated url.
The old one
https://agroportal.lirmm.fr/ontologies/PBO
does indeed show the new one, but when I try it, it redirects to
http://www.berkeleybop.org/ontologies/PBO
which then shows a 404 error.
Likewise the urlPrefix.
The old ones appear to be working though.

Reply:
Good catch 🙈 This is now fixed in [c78491b](https://github.com/tripal/tripal/pull/2288/commits/c78491b6a13b829f27bbbdd9e3d3fcbdf5605c98) which uses the old URL and URL prefix in both places.